app/src/logging/__init__.py

```diff
@@ -11,7 +11,8 @@
 Usage:
     import src.logging
 
-    src.logging.init("program name")
+    with src.logging.Log("program name"):
```

Comment:
Nit: Purely stylistically it'd be nice to be consistent with the pattern that there's an all lower case function that developers call that return a context manager, rather than instantiating the context manager itself. e.g.

`engine.connect()` returns `Connection`, `open` returns a `File`, etc., so could we still have a wrapper function like `init`/`init_logging` or `configure`/`configure_logging` that just returns the context manager? i.e.

```python
def init(program_name: str) -> LoggingContext:
    return LoggingContext(program_name)
```